### PR TITLE
Start deleting docs

### PR DESCRIPTION
**Pull request by liamsi**
- [x] add proper readme
- [x] deduplicate architecture + lazy-adr dirs
- [x] go through adr template and update it accordingly
- [x] update the adr readme if necessary

ref: #207

### CODE REVIEW

**Comment by liamsi** [comment]
image

image

File: docs/README.md [hunk 1]
```diff
@@ -1,33 +1,22 @@
----
-title: Tendermint Core Documentation
-description: Tendermint Core is a blockchain application platform.
-footer:
-  newsletter: false
----
+# Tendermint and LazyLedger
```
Comment (liamsi) [inline docs/README.md:1]: Rendered: https://github.com/lazyledger/lazyledger-core/tree/ismail/sweep_docs/docs#tendermint-and-lazyledger

**Review by marbar3778 — APPROVED** [review]

File: docs/README.md (hunk 1 above)
Comment (liamsi) [reply docs/README.md:1]: OK, I'll merge it now but I'd like @adlerjohn to have a look at the rendered version with his [Adleraugen](https://worldofdictionary.com/dict/german-english/meaning/adlerauge) :-)

Comment (adlerjohn) [reply docs/README.md:1]: Looks fine to me!

**Pull request merged**